packages/mermaid/src/docs/index.md

```diff
@@ -11,27 +11,31 @@ hero:
   tagline: JavaScript based diagramming and charting tool that renders Markdown-inspired text definitions to create and modify diagrams dynamically.
 
   image:
-    src: /mermaid-logo.svg
-    alt: Mermaid
+    light: /Hero.svg
+    dark: /Hero-Dark.svg
+    alt: Mermaid diagram
   actions:
     - theme: brand
-      text: Get Started
-      link: /intro/
+      text: Try Playground
+      link: https://www.mermaidchart.com/play
```

Comment:
Need to notify users that they're being redirected away to a different commercial product. This should be differentiated to show that. Maybe a logo, icon, or a popup.